components/brave_wallet_ui/common/hooks/swap.ts

```diff
@@ -142,6 +142,8 @@ export default function useSwap (
       default:
         return 'unknownError'
     }
+
+    return undefined
```

Comment:
@onyb - the new eslint disallows empty returns and our ts complains on how all code paths are not reachable. We basically don't allow implicit returns. Marked this one off explicitly.